bootstrap/pkg/apis/apps/configconverters/v1beta1.go

```diff
@@ -12,6 +13,30 @@ import (
 type V1beta1 struct {
 }
 
+func isPlatform(pluginKind string) bool {
+	platforms := map[string]bool{
+		string(kfconfig.AWS_PLUGIN_KIND): true,
+		string(kfconfig.GCP_PLUGIN_KIND): true,
+	}
+
+	_, ok := platforms[pluginKind]
+	return ok
+}
+
+func maybeGetPlatform(pluginKind string) string {
+	platforms := map[string]string{
```

Comment:
(minor) Should we just use switch?

Reply:
I don't know, which one is a better solution in golang?